examples/misc/lib/effective_dart/docs_good.dart

```diff
@@ -259,10 +280,10 @@ class ToggleComponent {}
 
 // #docregion this
 class Box {
-  /// The value this wraps.
+  /// The value this box wraps.
   Object? _value;
 
-  /// True if this box contains a value.
+  /// Whether this box contains a value.
```

Comment:
```suggestion
  /// Checks if this box contains a value.
```

Reply:
Yes, this is technically a function but it's a property, but developers shouldn't need to care about the code it runs, only what the property it provides access to is.

Reply:
Agree, and my primary point of this CL: It's a property. It's already recognized and named as one, so it should be documented as one too. And a boolean property would use `Whether`.

Also "Checks if" doesn't say what the result of that check is used for. With that doc, it could also be a `void` founction that throws if the check fails. It would have to be followed by a `Returns ...`, and that would likely be `Returns whether this box contains a value.`, which duplicates almost all of the first line.

Reply:
Got it!